pulp_smash/utils.py

```diff
@@ -149,3 +149,33 @@ def setUpClass(cls):
         stack.callback(testcase.tearDownClass)
         yield stack
         stack.pop_all()
+
+
+def get_pulp_setting(cli_client, setting_name, python_type="str"):
```

Comment:
What about calling something like this?
`pulpcore-manager shell -c "from django.conf import settings; print(repr(settings.INSTALLED_PULP_PLUGINS))"`

Reply:
That's a good idea. I'll change it to `pulpcore-manager shell -c "from django.conf import settings; print(repr(getattr(settings, setting_name, None)))"`

Reply:
I am trying this on an older container (3.6.2) and the output has some extra text in it. 

```
pulp: pulpcore.app.serializers.base:WARNING: Please provide either 'view_name' or 'view_name_pattern' for DetailRelatedField on _call_with_frames_removed.
'pulpcore.app.models.storage.FileSystem'
```

Reply:
Hmm, the warning should be sent to stderr, right? You should be able to filter that.